packages/terra-switch/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+ChangeLog
+=========
+
+Unreleased
+----------
+
```

Comment:
nit: this extra line should be removed

Reply:
updated changelog : 48fce2d9ca73773ac6f9891feaa7f2a56b36f864